reactivesocket-cpp/src/ReactiveStreamsCompat.h

```diff
@@ -2,7 +2,7 @@
 
 #pragma once
 
-#include "reactive-streams-cpp/ReactiveStreams.h"
+#include <ReactiveStreams.h>
```

Comment:
While I understand @benjchristensen argument that you can still have collisions with subfolders, I would say that putting one header file into the root folder is just asking for the collision earlier. I am used to thinking that if I see #include "file.h" then the header file should be picked up from the same folder as the cpp file being compiled.
I am also hesitant to say that we will "always" use only one header file. I broke my own promises of this kind too many times in the past.